### DISCUSSION TITLE
Add Row Layout in Table field like in Super Table plugin

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/cms#13227] [Ideas]
Hello,
when there are many columns (10 or more) in Table field there is problem with them fitting on screen.
It could be improved by adding Row Layout option like in Super Table plugin.
Thanks.

**Comment by thupsi** [comment]
I think this would come in handy, especially  when using a table field as a static (not repeating) "field group". You could use a matrix for this but it adds complexity and the UI is not compact.

Also it will help to have this layout option when relations land in table fields in Craft 5.

**Reply by brandonkelly** [reply]
If each of the columns can have consistent value types, the enhancements in #13231 should work for you.

(Not ruling out the ability to change the table’s orientation though.)

**Reply by thupsi** [reply]
That's a great enhancement as usual Brandon, thanks! But, yes, flipping the orientation would still benefit certain use cases.
